_rules/text-spacing-not-important-b8bb68.md

```diff
@@ -0,0 +1,264 @@
+rule_type: atomic
+description: |
+  This rule checks that the text spacing styles can be adjusted by user, i.e. that `!important` is not applied to the style, unless the style has already exceeded the specified metric for retaining content visibility and functionality.
+accessibility_requirements:
+  wcag21:1.4.12: # Text Spacing (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+This rule applies to any HTML element that is [visible][] and has a [declared][] value for any of the following CSS properties:
+
+- [word-spacing][],
+- [letter-spacing][],
+- [line-height][];
+  which are specified via the `style` attribute.
+
+## Expectation 1
+
+For the test target's [word-spacing][] property, either:
+
+- it has no [declared][] value; or
+- its [declared][] value specified via the `style` attribute does not have the [important flag][]; or
+- its [computed][] value is at least `0.16` times its [computed][] [font-size][].
+
+## Expectation 2
+
+For the test target's [letter-spacing][] property, either:
+
+- it has no [declared][] value; or
+- its [declared][] value specified via the `style` attribute does not have the [important flag][]; or
+- its [computed][] value is at least `0.12` times its [computed][] [font-size][].
+
+## Expectation 3
+
+For the test target's [line-height][] property, either:
+
+- it has no [declared][] value; or
+- its [declared][] value specified via the `style` attribute does not have the [important flag][]; or
+- its [computed][] value is at least `1.5` times its [computed][] [font-size][].
+
+## Assumptions
+
+If there is a mechanism available on the page by which text spacing can be adjusted, failing this rule might not mean [success criterion 1.4.12 Text spacing](https://www.w3.org/TR/WCAG21/#text-spacing) is not satisfied.
+
+## Accessibility Support
+
+While some assistive technologies are able to set [user origin][] styles, others, such as browser extensions, are only able to set style with the [author origin][]. Such assistive technologies can not create styles with a high enough [priority][] to override a `style` attribute with the [important flag][]. If accessibility support does not include assistive technologies that override text spacing through [author origin][], this rule should not be used.
+
+## Background
+
+- [Understanding Success Criterion 1.4.12: Text Spacing](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html)
+- [CSS Text Module Level 3 - Spacing](https://www.w3.org/TR/css-text-3/#spacing)
+- [CSS Visual formatting model details](https://drafts.csswg.org/css2/visudet.html)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `p` element has a `line-height` of `32px` (equals `2em` ) which is above the recommended metric, given the default pixel size is 16 pixels.
```

Comment:
There is no "default pixel size" in the specs. That is why the example specifies it.
This applies to the descriptions of all passed and failed examples.

```suggestion
This `p` element has a `line-height` of `32px` (equals `2em` ) which is above the recommended metric, given the specified font size is 16 pixels.
```